### PR TITLE
Basic SLEPOS test

### PR DESCRIPTION
**Pull request by nadvornik**

### CODE REVIEW

File: tests/slepos/prepare.pm [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+
+
+    # let's see how it looks at the beginning
+    save_screenshot;
+
+    # init
+    select_console 'user-console';
+
+    become_root;
```
Comment (aaannz) [inline tests/slepos/prepare.pm]: why are you switching to `user-console` and then becoming root? Why not `select_console 'root-console'`?

**Comment by nadvornik** [comment]
you are right, select_console is now fixed

**Comment by aaannz** [comment]
LGTM

**Comment by coolo** [comment]
please schedule it in production

**Comment by coolo** [comment]
(aehm, if we had SLEPOS in testing that is :)

**Pull request merged**